pkg/cloud/ovirt/machine/actuator.go

```diff
@@ -376,10 +376,79 @@ func (actuator *OvirtActuator) reconcileConditions(
 	return conditions
 }
 
+
+// validateMachine execute several validations, example: check if user provided in
```

Comment:
validateMachine validates the machine object yaml fields and returns InvalidMachineConfiguration in case the validation failed